backend/tests/test_lifespan.py

```
@@ -22,94 +22,67 @@ def test_app():
-    ) as mock_ensure_containers, patch(
-        "backend.api.stop_containers", side_effect=shutdown_error
-    ):
-
-        async with lifespan(test_app):
-            mock_ensure_containers.assert_called_once()
-            mock_logger.info.assert_any_call("Starting application containers...")
-
-        mock_logger.error.assert_called_with(
-            f"Error during container shutdown: {shutdown_error}"
-        )
+from backend.api import lifespan
```

Ruff (0.8.2)

Redefinition of unused `lifespan` from line 7

Remove definition: `lifespan`

(F811)